GTViewController/Classes/Custom Classes/UISnuffleButton.m

```diff
@@ -445,15 +454,19 @@ - (void)touchesEnded:(NSSet *)touches withEvent:(UIEvent *)event {
 				
 			}
 			
-            [self.tapEvents enumerateObjectsUsingBlock:^(NSString *eventName, NSUInteger idx, BOOL *stop) {
-                if (eventName) {
-                    NSString *trimmedEventName = [eventName stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceCharacterSet]];
-                    
-                    [[NSNotificationCenter defaultCenter] postNotificationName:UISnuffleButtonNotificationButtonTapEvent
-                                                                        object:self
-                                                                      userInfo:@{UISnuffleButtonNotificationButtonTapEventKeyEventName: trimmedEventName}];
-                }
-            }];
+            if (self.forceValidation && self.tapDelegate && [self.tapDelegate respondsToSelector:@selector(didReceiveTapOnPositiveButton:)]) {
```

Comment:
There are a couple advantages I see to keeping it this way:
 - the logic for sending the followup:subscription is all handled w/in `GTFollowupViewController` and therefore reduces a notification that has to be observed and sent. *primary benefit*
 - the method name `didReceiveTapOnPositiveButton` follows the pattern of other delegate method names. *tiny, secondary benefit*